mineral-ui/mineral-ui
# [ButtonGroup]: Fix support for styled/themed Button children

### Description

Fixes support for styled/themed Button children

### Motivation and context

Issue reported by user in chat room and illustrated in the following example.

https://codesandbox.io/s/4zvyzy3694

### Screenshots, videos, or demo, if appropriate

A "children" example has been added to the ButtonGroup page, but it is dev-only and won't show up on Netlify.

https://fix-button-group--mineral-ui.netlify.com/components/button-group

### How to test

1. Check the Happo diff?
2. Checkout the branch locally, and view ButtonGroup's "children" example.
3. Note that the there are no errors in the console, and that colors are appropriate, particularly variants, when styled/themed and used inside of Dropdown

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist
